phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexScrutinyToolIT.java

```diff
@@ -32,914 +31,621 @@
-            int numRows = count.getInt(1);
-            return numRows;
-        }
-
-    }
+    private static final String DELETE_SQL = "DELETE FROM %s ";
 
-    @RunWith(Parameterized.class) public static class IndexScrutinyToolNonTenantIT extends SharedIndexToolIT {
+    private String schemaName;
+    private String dataTableName;
+    private String dataTableFullName;
+    private String indexTableName;
+    private String indexTableFullName;
 
-        private String dataTableDdl;
-        private String indexTableDdl;
+    private Connection conn;
 
-        private static final String UPSERT_SQL = "UPSERT INTO %s VALUES(?,?,?,?)";
+    private PreparedStatement dataTableUpsertStmt;
 
-        private static final String
-                INDEX_UPSERT_SQL =
-                "UPSERT INTO %s (\"0:NAME\", \":ID\", \"0:ZIP\", \"0:EMPLOY_DATE\") values (?,?,?,?)";
+    private PreparedStatement indexTableUpsertStmt;
 
-        private static final String DELETE_SQL = "DELETE FROM %s ";
+    private long testTime;
+    private Properties props;
 
-        private String schemaName;
-        private String dataTableName;
-        private String dataTableFullName;
-        private String indexTableName;
-        private String indexTableFullName;
-
-        private Connection conn;
-
-        private PreparedStatement dataTableUpsertStmt;
-
-        private PreparedStatement indexTableUpsertStmt;
-
-        private long testTime;
-        private Properties props;
-
-        @Parameterized.Parameters public static Collection<Object[]> data() {
-            return Arrays.asList(new Object[][] { { "CREATE TABLE %s (ID INTEGER NOT NULL PRIMARY KEY, NAME VARCHAR, ZIP INTEGER, EMPLOY_DATE TIMESTAMP, EMPLOYER VARCHAR)",
-                    "CREATE LOCAL INDEX %s ON %s (NAME, EMPLOY_DATE) INCLUDE (ZIP)" }, { "CREATE TABLE %s (ID INTEGER NOT NULL PRIMARY KEY, NAME VARCHAR, ZIP INTEGER, EMPLOY_DATE TIMESTAMP, EMPLOYER VARCHAR) SALT_BUCKETS=2",
-                    "CREATE INDEX %s ON %s (NAME, EMPLOY_DATE) INCLUDE (ZIP)" }, { "CREATE TABLE %s (ID INTEGER NOT NULL PRIMARY KEY, NAME VARCHAR, ZIP INTEGER, EMPLOY_DATE TIMESTAMP, EMPLOYER VARCHAR) SALT_BUCKETS=2",
-                    "CREATE LOCAL INDEX %s ON %s (NAME, EMPLOY_DATE) INCLUDE (ZIP)" } });
-        }
+    @Parameterized.Parameters public static Collection<Object[]> data() {
+        return Arrays.asList(new Object[][] { { "CREATE TABLE %s (ID INTEGER NOT NULL PRIMARY KEY, NAME VARCHAR, ZIP INTEGER, EMPLOY_DATE TIMESTAMP, EMPLOYER VARCHAR)",
+                "CREATE LOCAL INDEX %s ON %s (NAME, EMPLOY_DATE) INCLUDE (ZIP)" }, { "CREATE TABLE %s (ID INTEGER NOT NULL PRIMARY KEY, NAME VARCHAR, ZIP INTEGER, EMPLOY_DATE TIMESTAMP, EMPLOYER VARCHAR) SALT_BUCKETS=2",
+                "CREATE INDEX %s ON %s (NAME, EMPLOY_DATE) INCLUDE (ZIP)" }, { "CREATE TABLE %s (ID INTEGER NOT NULL PRIMARY KEY, NAME VARCHAR, ZIP INTEGER, EMPLOY_DATE TIMESTAMP, EMPLOYER VARCHAR) SALT_BUCKETS=2",
```

Comment:
I think we should be able to add "MULTI_TENANT=TRUE" without anything failing.  But I guess "testColumnsForSelectQueryOnMultiTenantTable" in the other class already tests what we want.

Reply:
yeah, the multitenant_tenant = true ones are in the other file